core/math/convex_hull.cpp

```diff
@@ -1775,7 +1775,7 @@ real_t ConvexHullInternal::shrink(real_t p_amount, real_t p_clamp_amount) {
 	return p_amount;
 }
 
-bool ConvexHullInternal::shift_face(Face *p_face, real_t p_amount, LocalVector<Vertex *> p_stack) {
+bool ConvexHullInternal::shift_face(Face *p_face, real_t p_amount, LocalVector<Vertex *> &p_stack) {
```

Comment:
The passed vector is always empty (and left empty that the end of the method) so this didn't actually copy anything, but I am pretty sure that the intention (and probably function in the original bullet code) is to avoid allocations by sharing the stack